trie/utils/verkle_test.go

```diff
@@ -113,3 +113,20 @@ func TestCompareGetTreeKeyWithEvaluated(t *testing.T) {
 		}
 	}
 }
+
+func BenchmarkGetTreeKeyWithEvaluatedAddress(b *testing.B) {
+	var buf [32]byte
+	rand.Read(buf[:])
+	addrpoint := EvaluateAddressPoint(buf[:])
+
+	rand.Read(buf[:])
+	n := uint256.NewInt(0).SetBytes32(buf[:])
+
+	_ = verkle.GetConfig()
+
+	b.ReportAllocs()
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		_ = GetTreeKeyWithEvaluatedAddess(addrpoint, n, 0)
+	}
+}
```

Comment:
Out of curiosity I added this benchmark to compare the pre and post change, and didn't notice any significant speedup.

In any case, avoiding an auxiliary array and copying is good both for doing less work, and also a more compact implementation. (and, if you mind, a closer match to `GetTreeKey(...)` since there's no reason why doing the same thing in different ways).